script/e2e-test.sh

```diff
@@ -18,6 +18,7 @@ DOCKER_PASSWORD=${6:-""}
 TEST_TIMEOUT_MINUTES=${7:-4}
 DEX_IP=${8:-"172.18.0.2"}
 ADDITIONAL_CLUSTER_IP=${9:-"172.18.0.3"}
+KAPP_CONTROLLER_VERSION="v0.32.0"
```

Comment:
Mmm, perhaps we should pass this var also from the circleci config, like we already do for the `OLM_VERSION` ?

Reply:
Woops, yes. Forgot to update it. Done.